components/patient-data/ui/src/main/resources/PhenoTips/UIX_Field__charts.xml

```diff
@@ -116,9 +116,13 @@
 #set ($targetClassName = 'PhenoTips.MeasurementsClass')
 #if ($xcontext.action != 'export' &amp;&amp; ($xcontext.action == 'edit' || $doc.getObject($targetClassName)))
   (% class="growth-charts-section" %)(((
-    === $services.localization.render('phenotips.UIXField.growthCharts.heading') ===
-
-    #__expandTools()
+    #if($xcontext.action != 'view')
```

Comment:
That's not correct, there are more options than `view` and `edit`, and the only one where it makes sense to have it is in edit mode. For example, there's also `export`, and we don't want to have the `hide` button in the PDF.

Also, leave a space in `#if (`.